scripts/dev/e2e.py

```diff
@@ -45,14 +45,12 @@ def _prepare_test_environment(config_file: str) -> None:
 
     print("Creating Role")
     k8s_conditions.ignore_if_already_exists(
-        lambda: rbacv1.create_namespaced_role(dev_config.namespace, _load_test_role())
+        lambda: rbacv1.create_cluster_role(_load_test_role())
```

Comment:
Needed to use a cluster role to be able to create a cluster role for the clusterwide test.